js/ElementRendererProvider.js

```diff
@@ -357,9 +358,17 @@ export default class ElementRendererProvider {
       if (config.style) {
         divEl.style.cssText = Utils.styleToCss(config.style);
       }
-
-      if (config.accessibility && config.accessibility.web) {
+  
+      if (accessibilityWeb) {
         Utils.appendAttributesFromObject(divEl, config.accessibility.web);
+      } else if ((accessibilityWeb && !config.accessibility.web.tabindex) || !config.accessibility) {
```

Comment:
@FeelsGOOD can you explain your approach with the if statement here?

Reply:
@kelvingraddick  Based on my understanding, i think we are adding default tabindex="0" for maps if there is no external accessibility json in the structured content. Can you confirm @FeelsGOOD